packages/genesys-spark-components/src/components/stable/gux-tooltip/gux-tooltip.tsx

```diff
@@ -131,7 +131,10 @@ export class GuxTooltip {
   private updatePosition(): void {
     const middleware = [
       offset(12),
-      flip(),
+      flip({
+        fallbackAxisSideDirection: 'start',
+        crossAxis: false
```

Comment:
These settings were required as when the screen is small it was not detecting overflow. With these options added it will flip like so `[ 'right', 'left', 'top' , 'bottom']` . Notes are here  (https://github.com/floating-ui/floating-ui/pull/2082#issuecomment-1368384210)

Also on the docs website : https://floating-ui.com/docs/flip#fallbackaxissidedirection

Reply:
I dont know if we should apply this to the `tooltip-beta` as it does not really apply to it yet ?

Reply:
I think we should keep the beta component in sync because something like this will be easily missed when we eventually transition `gux-truncate` to use `gux-tooltip-beta'.